src/components/utils/FormContainer/FormContainer.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import * as classes from "./FormContainer.module.scss";
+import CustomInput from "../CustomInput/CustomInput";
+import Button from "../Button/Button";
+import propTypes from "prop-types";
+
+const FormContainer = (props) => {
+  const [isValid, setIsValid] = useState(false);
+  const [isValidated, setIsValidated] = useState(false);
+  const [isSending, setIsSending] = useState(false);
+  const [isBtnDisabled, setIsBtnDisabled] = useState(false);
+
+  const validateInput = (e) => {
+    const value = document.querySelector(`#${props.inputID}`).value;

```

Comment:
This is bad, you should avoid `document.querySelector` calls (it is slow). You should use react APIs.